components/automate-chef-io/content/docs/desktop.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Desktop Dashboard"
+description = "Using the Desktop Dashboard"
+date = 2018-03-26T16:01:47-07:00
+draft = false
+bref = ""
+toc = true
++++
+
+The Chef Automate _Desktop_ dashboard displays status information about all desktops connected to Chef Automate.
+Desktop information populates this dashboard after a Chef Infra Client run has executed.
+
+Node counts in the _Desktop_ dashboard may include liveness agents. Desktop information appears in a series of displays. The _Desktop_ dashbaord does not support project filtering.
+
+## Setting Up The Desktop Dashboard
+
+Include a `--product desktop` flag to enable the Desktop dashboard when [installing Chef Automate](https://automate.chef.io/docs/infra-server/). The Desktop dashboard has no supported compliance profiles, and no compliance profiles will be included in installation when using the `--product desktop` flag.
```

Comment:
@vjeffrey @susanev I used this link for installation reference per our previous comment thread.

Reply:
um, why are we linking to the automate chef server install? is it bc that install also uses the --product flag?

Reply:
we should prolly link to config, but also i think @yzl has a pr to add a section to config about adding products.. 🤷

Reply:
We should give the full command (maybe we do?) The command I was given  last week is: `chef-automate deploy --product automate --product infra-server --product desktop`. Is that correct?  The automate + infra install docs are the most comprehensive for system requirements and installation methods.

Reply:
we do not need the --product automate part. we only need the chef server part if they're also planning on doing the chefserver/automate on the same box, and not using a sep chef server

Reply:
So then sending them to the different ways to install Chef Infra Server and/or Chef Automate together doc is a relevant link...?